capture/src/main/kotlin/com/alfresco/capture/CameraFragment.kt

```diff
@@ -72,6 +72,9 @@ class CameraFragment : Fragment(), KeyHandler, MavericksView {
         lifecycleScope.launch {
             updateCameraState()
         }
+
+        layout.modeSelectorView.retainLastState(viewModel.mode)
+        setMode(viewModel.mode)
```

Comment:
given that setMode does just one thing, i.e. configure the shutter button i think it's bad use of it.
either group multiple mode related settings or not use it at all

Reply:
Done